documentation/book/appendix_crds.adoc

```diff
@@ -1885,6 +1887,23 @@ It must have the value `transactionalId` for the type `AclRuleTransactionalIdRes
 |string (one of [prefix, literal])
 |====
 
+[id='type-KafkaUserQuotas-{context}']
+### `KafkaUserQuotas` schema reference
+
+Used in: xref:type-KafkaUserSpec-{context}[`KafkaUserSpec`]
+
+
+[options="header"]
+|====
+|Property                  |Description
+|consumerByteRate   1.2+<.<|This quota is defined on a per-broker basis. Each group of clients can fetch a maximum of X bytes/sec per broker before clients are throttled.
+|integer
+|producerByteRate   1.2+<.<|This quota is defined on a per-broker basis. Each group of clients can publish a maximum of X bytes/sec per broker before clients are throttled.
+|integer
+|requestPercentage  1.2+<.<|CPU utilization thresholds as a percentage of network and I/O threads.
```

Comment:
_Please reword:_

Sets a quota on the maximum CPU utilization of each client group as a percentage of network and I/O threads.